scripts/clippy_wasm.sh

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env bash
-# This scripts run clippy on the wasm32-unknown-unknown target,
-# using a special clippy.toml config file which forbids a few more things.
-
-set -eu
-script_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
-cd "$script_path/.."
-set -x
-
-# Use scripts/clippy_wasm/clippy.toml
-export CLIPPY_CONF_DIR="scripts/clippy_wasm"
-
-cargo install --locked --quiet cargo-cranky
-
-cargo cranky --all-features --target wasm32-unknown-unknown --target-dir target_wasm -p re_viewer -- --deny warnings
```

Comment:
This functionality must remain! We check for important stuff here, like not using `std::time::SystemTime::now` anywhere

Reply:
it moved into rust_checks and runs on main and nightly

Reply:
CLIPPY_CONF_DIR was missed though!